holidays/countries/__init__.py

```diff
@@ -89,7 +89,7 @@
 from .slovenia import Slovenia, SI, SVN
 from .south_africa import SouthAfrica, ZA, ZAF
 from .spain import Spain, ES, ESP
-from .swaziland import Swaziland, SZ, SZW
+from .eswatini import Eswatini, Swaziland, SZ, SZW
```

Comment:
It seems that the imports need to be sorted alphabetically.

Reply:
Done.